### PR TITLE
main chore: release notes single-sourcing script fixes and enhancements

### PR DESCRIPTION
**Pull request by themr0c**
*    Uses the attributes.adoc file to retrieve product version (y and z), rather than the YAML file that we forgot to update.
*    Fixed the Jira search to be accurate on a z-stream. (The script was tested on a y-stream only, and had weird results on a z-stream).
*    Uses only one content template, rather than 3.
*    Generates only 6 files, rather than one file per Jira.
*    In the Fixed issues section create one subtitle for each z-stream version.
*     Double checked that the Jira searches are accurate.
* handle the case when the z-stream is 0.

**IMPORTANT: Do Not Merge - To be merged by Docs Team Only**

**Version(s):** 1.5

Add the relevant labels to the Pull Request.
**Issue:**

### CODE REVIEW

File: modules/release-notes/single-source-release-notes.py [hunk 1]
```diff
@@ -13,109 +13,93 @@
...
+with open(attributes_file) as file:
+  attributes = {}
+  for lines in file:
+    if re.search(':', lines):
+      items = lines.rsplit(':', 2)
+      attributes[items[1]] = items[2].strip()
+
+product_version_minor = attributes["product-version"]
+product_version_patch = attributes["product-bundle-version"]
+
```
Comment (themr0c) [inline modules/release-notes/single-source-release-notes.py:52]:
```suggestion
# Until 1.4 at least, Jira has no z-stream 0. Use the y-stream instead.
if re.search('.0$',product_version_patch):
  product_version_patch = product_version_minor

```

**Review by Fortune-Ndlovu — COMMENTED** [review]
@themr0c Looks good, can you give me context as to why these enhancements are needed?

**Comment by themr0c** [comment]
@Fortune-Ndlovu I updated the PR description to answer your question.

File: artifacts/attributes.adoc [hunk 2]
```diff
@@ -10,9 +10,9 @@
 :product: Red Hat Developer Hub
 :product-short: Developer Hub
 :product-very-short: RHDH
-:product-version: 1.4
-:product-bundle-version: 1.4.0
-:product-chart-version: 1.4.0
+:product-version: 1.5
+:product-bundle-version: 1.5.0
+:product-chart-version: 1.5.0
 :product-backstage-version: 1.29.2
```
Comment (themr0c) [inline artifacts/attributes.adoc:16]:
```suggestion
:product-backstage-version: 1.31.3
```

**Pull request merged**